### ISSUE TITLE
Coded log-in design for website (not functional)

### ISSUE THREAD

**Issue by catherineunciano** [IT3A-2016/Paper-Icebergs#27]

**Comment by giomigueltada** [comment]
![image](https://zube.io/files/IT3A-2016/b8d5f2767ba22b762ca0b38b8a877740.png)

**Comment by catherineunciano** [comment]
<!DOCTYPE html>
<html>
	<head>
	
		<title>Queueing Web System</title>
		
		<link rel="stylesheet" type="text/css" href="../css/bootstrap.css">
		<meta name ="viewport" content="width=device-width, initial-scale-1">
		
		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
		
	</head>
<body>
	<div class="container" style="padding-top: 30px;">
			<div class="row">
				<div class="col-sm-12 col-md-12 col-lg-12 col-xs-12"><br>
					<div class="font-family">
						<div class="text-center login-title"><h1>Login</h1><br>
							<form class="form-signin" action="" method="post">
							<input id="username" name="username" type="text" class="form-control" placeholder="Email" required autofocus><br>
							<input id="password" name="password" type="password" class="form-control" placeholder="Password" required><br>
							<a href="login.php"><center>
							<button class="btn btn-lg btn-primary btn-block" type="submit">
							<font color="black">Submit</center></button></font><br></a>
							<a href="#"><font color="#ffffff;"><br>Forgot your password?</a>
							<a href="signup.php" class="text-center new-account">Create an account </a></font>
							</form>
						</div>
					</div>
				</div>
			</div>
	</div>
</body>
</html>

**Comment by catherineunciano** [comment]
<!DOCTYPE html>
<html>
	<head>
	
		<title>Queueing Web System</title>
		
		<link rel="stylesheet" type="text/css" href="../css/bootstrap.css">
		<meta name ="viewport" content="width=device-width, initial-scale-1">
		
		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
		
	</head>
<body>
	<div class="container" style="padding-top: 30px;">
			<div class="row">
				<div class="col-sm-12 col-md-12 col-lg-12 col-xs-12"><br>
					<div class="font-family">
						<div class="text-center login-title"><h1>Login</h1><br>
							<form class="form-signin" action="" method="post">
							<input id="username" name="username" type="text" class="form-control" placeholder="Email" required autofocus><br>
							<input id="password" name="password" type="password" class="form-control" placeholder="Password" required><br>
							<a href="login.php"><center>
							<button class="btn btn-lg btn-primary btn-block" type="submit">
							<font color="black">Submit</center></button></font><br></a>
							<a href="#"><font color="#ffffff;"><br>Forgot your password?</a>
							<a href="signup.php" class="text-center new-account">Create an account </a></font>
							</form>
						</div>
					</div>
				</div>
			</div>
	</div>
</body>
</html>
